docs/index.src.html

```diff
@@ -3092,12 +3092,13 @@ <h1 id="api-json">JSON</h1>
 
     <h1 id="api-macro">Macro</h1>
 
-    <p>You may be familiar with the macro concept if you are used to working with Laravel and objects such as <a
-            href="https://laravel.com/docs/5.6/responses#response-macros">response</a> or <a
-            href="https://laravel.com/docs/5.6/collections#extending-collections">collections</a>. The Carbon <code>macro()</code>
-        works the exact same way as the Laravel MacroableTrait one, it takes a method name as first argument and a
-        closure as second. This will make the closure action available on all Carbon instances (and also as a Carbon static
-        method) as a method with the given name.</p>
```

Comment:
We should keep "as a method with the given name".

About static method, it can be interesting to create macros that are both static and dynamic, like:
```php
Carbon::macro('doubleYear', function () {
  if (isset($this)) {
    return $this->year * 2; // If called dynamically
  }

  return Carbon::now()->year * 2; // If called statically
});

Carbon::parse('1990')->doubleYear(); // 3980
Carbon::doubleYear(); // 4038
```

So I'm not sure we should un-document it. For me the `static function` is just a way to make it more explicit (and help static analyzers) but every macros (with the `static` keyword or not) will always be available both dynamically and statically until Laravel would decide to change it making the `static` keyword mandatory and using 2 different variables for `__call` available macros and `__callStatic` available macros. For now, Laravel and Carbon both use a single common variable.